src/site/content/en/blog/building-multiple-pwas-on-the-same-domain/index.md

```diff
@@ -0,0 +1,264 @@
+`https://calendar.example.com`, while offering the main service of their
+business at `https://www.example.com`. Another example is a sports site that
+wants to create an independent app, completely dedicated to an important sports
+event, like a football championship at `https://footballcup.example.com`, that
+users can install and use independently of the main sport site, hosted at
+`https://www.example.com`. This approach might also be useful for platforms that
+let customers create independent apps of their own under the company's brand.
+For example, an app that lets merchants create their own PWAs at
+`https://merchant1.example.com`, `https://merchant2.example.com`, etc.
+
+Using different origins ensures isolation between the apps, which means that
+each of them will manage different browser features independently, including:
+
+- **Installability:** Each app has its own Manifest and provides its own
+  installable experience.
+- **Storage:** Each app has its own caches, local storage, and basically all
+  forms of device-local storage, without sharing them with the others.
+- **Service Workers:** Each app has its own service worker(s) for the registered
+  scopes.
+- **Permissions:** Permissions are also scoped by origins. Thanks to that, users
+  will know exactly for which service they are giving permissions for, and
+  features like notifications will be properly attributed to each app.
+
+Creating such a degree of isolation is the most desirable in the use case of
+multiple independent PWAs, so **we strongly recommend this approach**.
+
+If apps on subdomains want to share local data with each other they will still
+be able to do it via **cookies**, or for more advanced scenarios they could
+consider synchronizing the storage through a server.
+
+<figure class="w-figure">
+  {% Img src="image/26V1DWN36MZr3mUo8ChSBlCpzp43/FfuseK64y4PRqXYEGEt9.png", alt="ALT_TEXT_HERE", width="800", height="421" %}
+  <figcaption class="w-figcaption">Building different PWAs in distinct origins, by using subdomains is a good practice.</figcaption>
+</figure>
+
+## Using the same origin
+
+The second approach is building the different PWAs on the same origin. This
+includes the following scenarios:
+
+### Non-overlapping paths
+
+Multiple PWAs or conceptual "web apps", hosted on the same origin, with
+non-overlapping paths. For example:
+
+- `https://example.com/app1/`
+- `https://example.com/app2/`
+
+### Overlapping/nested paths
+
+Multiple PWAs on the same origin, one of whose scope is nested inside the other:
+
+- `https://example.com/` (the "outer app")
+- `https://example.com/app/` (the "inner app")
+
+The service worker API and manifest format allow you to do either of the above,
+using path-level scoping. However, in both cases, using the same origin presents
+many problems and limitations, the root of which stems from the fact that the
+browser won't fully consider these to be distinct "apps", therefore **this
+approach is discouraged**.
```

Comment:
I understand that in most cases we tend to talk about the "positive" case and avoid talking too much about the "negative" one and just say that it's not recommended. 
In this particular article, talking about the non-recommended use case is important because is a very common mistake developers do, and they don't really understand why it's a mistake.

Let me know if that makes sense.

cc // @mgiuca